private_dot_claude/CLAUDE.md

```
@@ -132,3 +132,52 @@ const llmClient = createLLMClient(...);
+
+以下には **適用しない**:
+
+- 単一ファイルの軽微な修正
+- ドキュメントのみの変更
+- 設定値の変更
+
+### 手順
+
+1. プランファイルに実装計画を書き終えたら、Codex にレビューを依頼する:
+
+```bash
```

markdownlint-cli2 (0.21.0)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)